src/lib/djerba/plugins/tar/sample/plugin.py

```diff
@@ -125,22 +150,22 @@ def render(self, data):
     def process_ichor_json(self, ichor_metrics):
         with open(ichor_metrics, 'r') as ichor_results:
             ichor_json = json.load(ichor_results)
-        return(ichor_json)
+        return (ichor_json)
 
-    def process_consensus_cruncher(self, consensus_cruncher_file):
+    def process_consensus_cruncher(self, consensus_cruncher_file ):
         header_line = False
         with open(consensus_cruncher_file, 'r') as cc_file:
             reader_file = csv.reader(cc_file, delimiter="\t")
             for row in reader_file:
                 if row:
-                    if row[0] == "BAIT_SET" :
+                    if row[0] == "BAIT_SET":
                         header_line = True
                     elif header_line:
-                        unique_coverage = float(row[9]) 
+                        unique_coverage = float(row[9])
                         header_line = False
                     else:
                         next
-        return(int(round(unique_coverage, 0)))
+        return (int(round(unique_coverage, 0)))
```

Comment:
Again, outermost parentheses are not needed -- can have ` return int(round(unique_coverage, 0))`